### DISCUSSION TITLE
Why `Pod` doesn't require `Send` and `Sync`?

### DISCUSSION THREAD

**Discussion by Dzejkop** [Lokathor/bytemuck#273] [General]
I was doing some refactoring and found myself slapping `Pod + Send + Sync` all over the place. And it made me wonder - why would a type that's `Pod` also be `!Send` or `!Sync`?

**Comment by Lokathor** [comment]
Copy data is usually Send + Sync, but those traits weren't included just because they weren't strictly necessary for the design, and so no one has really mentioned it before.

**Reply by Dzejkop** [reply]
Fair, makes sense!
